src/windows/main_window.py

```diff
@@ -691,6 +702,8 @@ def actionImportFiles_trigger(self):
             start_location,
             )[0]
 
+        if len(qurl_list):
```

Comment:
what is this code for? `qurl_list`? Is there a reason we aren't just using our recommended_path from above?

Reply:
I figured the most logical `new default` path is the folder of the *last* file selected.

pseudocode:
```
if any files were selected
    setDefault <- <basename of LAST file selected>
```
I'll add a comment to explain that.